website/key.html

```diff
@@ -244,7 +243,7 @@ <h4>Mining key generation or input options:</h4>
     var printFrame = $id('printFrame');
     var printFrameDoc = printFrame.contentDocument || printFrame.contentWindow.document;
     printFrameDoc.body.innerHTML = '';
-    printFrameDoc.write('Private key hex data:<br> ' + wallet.getPrivateKeyHex() + '<script> function printPage(){ print() } <\/script>');
+    printFrameDoc.write('Private key hex data:<br /> ' + wallet.getPrivateKeyHex() + '<script> function printPage(){ print() } <\/script>');
```

Comment:
Part of my pathetic attempt to W3 validate page with https://validator.w3.org/ while debugging